ci/create-forklift-user-account.sh

```
@@ -111,27 +111,24 @@ bind_service_accont_to_role ${SERVICE_ACCOUNT}-user forklift-user
 setup_servie_account ${SERVICE_ACCOUNT}-reader
 bind_service_accont_to_role ${SERVICE_ACCOUNT}-reader forklift-reader
 
-# Print out tokens
-export TOKEN_ADMIN=$(kubectl get secret ${SERVICE_ACCOUNT}-admin -n ${NAMESPACE} -o=jsonpath={.data.token} | base64 -d)
-export TOKEN_USER=$(kubectl get secret ${SERVICE_ACCOUNT}-user -n ${NAMESPACE} -o=jsonpath={.data.token} | base64 -d)
-export TOKEN_READER=$(kubectl get secret ${SERVICE_ACCOUNT}-reader -n ${NAMESPACE} -o=jsonpath={.data.token} | base64 -d)
+# Write tokens to a restricted file instead of stdout
+TOKEN_ADMIN=$(kubectl get secret ${SERVICE_ACCOUNT}-admin -n ${NAMESPACE} -o=jsonpath={.data.token} | base64 -d)
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)